voter/index.md

```diff
@@ -864,47 +979,45 @@ import sdk "github.com/cosmos/cosmos-sdk/types"
 
 // BankKeeper defines the expected bank keeper
 type BankKeeper interface {
-	SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error
-	MintCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
-	BurnCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
-	SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
-	SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
+    SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error
+    MintCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
+    BurnCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
+    SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
+    SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
 }
 ```
 
-Second, add the keeper to the `x/voter/keeper/keeper.go` file.
-Add it to the `type` as well as the `NewKeeper` function as follows:
+Second, add the keeper to the `x/voter/keeper/keeper.go` file. Add it to the `type` and the `NewKeeper` function as follows:
 
 ```go
 type (
-	Keeper struct {
-		cdc        codec.Marshaler
-		storeKey   sdk.StoreKey
-		memKey     sdk.StoreKey
-		bankKeeper types.BankKeeper
-	}
+    Keeper struct {
+        cdc        codec.Marshaler
+        storeKey   sdk.StoreKey
+        memKey     sdk.StoreKey
+        bankKeeper types.BankKeeper
+    }
 )
 
 func NewKeeper(cdc codec.Marshaler, storeKey, memKey sdk.StoreKey, bankKeeper types.BankKeeper) *Keeper {
-	return &Keeper{
-		cdc:        cdc,
-		storeKey:   storeKey,
-		memKey:     memKey,
-		bankKeeper: bankKeeper,
-	}
+    return &Keeper{
+        cdc:        cdc,
+        storeKey:   storeKey,
+        memKey:     memKey,
+        bankKeeper: bankKeeper,
+    }
 }
 ```
 
-Finally, add the bank module to the `app.go` file in `x/app/app.go`
+Finally, add the bank module to the `app.go` file in `x/app/app.go`:
 
 ```go
 app.voterKeeper = *voterkeeper.NewKeeper(
   appCodec, keys[votertypes.StoreKey], keys[votertypes.MemStoreKey], app.BankKeeper,
 )
 ```
 
-Now you are ready to use all the bank functions that you added to the expected keepers file above.
-Next you will define how the transaction will require the funds in order to exectue the transaction.
+<!-- here is how far I got, but my app doesn't work yet --> Now, you are ready to use all the bank functions that you added to the expected keepers file above. Next, you need to define how the transaction will require the funds in order to exectue the transaction.
```

Comment:
@toschdev I really really wanted to succeed in this long-winded, very detailed tutorial... I got farther than I did before, but will benefit from some technical 1:1 

welcome back!